### PR TITLE
Add support for logos

### LINKED ISSUE #131: Add support for logos
## Description

EVA doesn't currently support logos, though they are available in emcpy.  These can be added in a similar fashion to `tight layout`, which was recently added.

## Requirements
Yaml files will be able to specify and plot logo(s -- nws and/or noaa) with appropriate options (location, size, etc).

## Dependencies

None

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
## Description

Extend `figure_driver.py` to parse logo information from yaml to emcpy in order to produce logo(s) in the resulting plots.

Note that in order to get the updated emcpy with the latest changes to `plot_logo` I bumped the emcpy commit index in `setup.py`.

## Dependencies

None.

## Impact

None.

Closes #131 .

### CODE REVIEW

File: src/eva/tests/config/testMonDataSpaceHirs4Metop-A.yaml [hunk 1]
```diff
@@ -26,7 +26,9 @@ graphics:
         figure size: [12,10]
         title: 'hirs4_metop-a | Channel 1 | Obs Count'
         output name: lineplots/hirs4_metop-a/hirs4_metop-a.0.count.png
-
+        plot logo:
```
Comment (CoryMartin-NOAA) [inline src/eva/tests/config/testMonDataSpaceHirs4Metop-A.yaml:29]: add logo? just a thought

Comment (kevindougherty-noaa) [reply src/eva/tests/config/testMonDataSpaceHirs4Metop-A.yaml:29]: That would require change to emcpy. Is that what you want?

Comment (CoryMartin-NOAA) [reply src/eva/tests/config/testMonDataSpaceHirs4Metop-A.yaml:29]: in that case then redacted, it's just semantics after all

**Comment by CoryMartin-NOAA** [comment]
looks like there is a conflict that needs resolved @EdwardSafford-NOAA

File: src/eva/tests/config/testMonDataSpaceHirs4Metop-A.yaml (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply src/eva/tests/config/testMonDataSpaceHirs4Metop-A.yaml:29]: Well the yaml could use "add logo" and then `figure_driver.py` could translate "add logo" to "plot_logo()" which is what emcpy uses.  No biggie.

**Comment by EdwardSafford-NOAA** [comment]
@CoryMartin-NOAA I think I've got the conflict fixed now.

**Review by CoryMartin-NOAA — APPROVED** [review]

**Comment by CoryMartin-NOAA** [comment]
@danholdaway @asewnath you may wish to add GMAO/NASA logos to the EMCPy repository to take advantage of this function, see https://github.com/NOAA-EMC/emcpy/pull/137

**Pull request merged**